packages/roosterjs-content-model-core/lib/command/createModelFromHtml/convertInlineCss.ts

```diff
@@ -28,16 +28,18 @@ export function retrieveCssRules(doc: Document): CssRule[] {
     const result: CssRule[] = [];
 
     styles.forEach(styleNode => {
-        const sheet = styleNode.sheet as CSSStyleSheet;
+        const sheet = styleNode.sheet;
```

Comment:
do we have a local repro? I wonder in what case this is null

Reply:
I do not have a local repro, but checking the telemetry, the error rate is really low. 

According to mozilla, this value can be null if there is no sheet associated with the element: 
A [StyleSheet](https://developer.mozilla.org/en-US/docs/Web/API/StyleSheet) object, or null if none is associated with the element.
https://developer.mozilla.org/en-US/docs/Web/API/HTMLStyleElement/sheet#value

Reply:
Looks like this is a way to repro:
![image](https://github.com/user-attachments/assets/c836bb63-a742-4bd4-9467-5219f8796bdf)